.github/workflows/cross-arch-test.yaml

```diff
@@ -0,0 +1,135 @@
+---
+name: Cross-Platform Build Tests
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  build-aix:
+    strategy:
+      matrix:
+        goarch: [ppc64]
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
+      - id: goversion
+        run: echo "goversion=$(cat .go-version)" >> "$GITHUB_OUTPUT"
+      - uses: actions/setup-go@d35c59abb061a4a6fb18e82ac0862c26744d6ab5 # v5.5.0
+        with:
+          go-version: ${{ steps.goversion.outputs.goversion }}
+      - name: Run golangci-lint
```

Comment:
@ivanvc i have added the linter to all the jobs as you proposed in https://github.com/etcd-io/bbolt/pull/993#issuecomment-2986987431